src/errors/test-run/templates.js

```diff
@@ -260,5 +260,9 @@ export default {
         Cannot evaluate the "${err.actualValue}" expression in the "${err.argumentName}" parameter because of the following error:
 
         ${err.errMsg}
+    `),
+
+    [TYPE.assertionUnawaitedPromiseError]: err => markup(err, `
+        Attempted to run assertions on a Promise object. Did you forget to await it? If not, pass "{ allowUnawaitedPromise: true }" to the assertion options.
```

Comment:
@VasilyStrelyaev, please, check the new error message

Reply:
@AndreyBelym It's okay